frontend/src/components/Worksheet/WorksheetStats.tsx

```diff
@@ -98,13 +105,59 @@ export default function WorksheetStats() {
                 </StyledStatPill>
               </li>
               <li>
-                <StyledStatPill>Total workload</StyledStatPill>
+                <StyledStatPill>
+                  Total workload
+                  {coursesWithoutWorkload.length > 0 && (
+                    <OverlayTrigger
+                      placement="top"
+                      overlay={(props) => (
+                        <Tooltip {...props} id="conflict-icon-button-tooltip">
+                          <small style={{ fontWeight: 500 }}>
+                            Computed with {coursesWithWorkload} course
+                            {coursesWithWorkload === 1 ? '' : 's'}.{' '}
+                            {coursesWithoutWorkload.length === 1
+                              ? coursesWithoutWorkload[0]
+                              : `${coursesWithoutWorkload.slice(0, -1).join(', ')} and ${coursesWithoutWorkload[coursesWithoutWorkload.length - 1]}`}{' '}
+                            ha
+                            {coursesWithoutWorkload.length > 1 ? 've' : 's'} no
+                            ratings.
+                          </small>
+                        </Tooltip>
+                      )}
+                    >
+                      <StyledInfo />
+                    </OverlayTrigger>
+                  )}
+                </StyledStatPill>
                 <StyledStatPill colormap={workloadColormap} stat={workload}>
                   {workload.toFixed(2)}
                 </StyledStatPill>
               </li>
               <li>
-                <StyledStatPill>Average rating</StyledStatPill>
+                <StyledStatPill>
+                  Average rating
+                  {coursesWithoutRating.length > 0 && (
+                    <OverlayTrigger
+                      placement="top"
+                      overlay={(props) => (
+                        <Tooltip {...props} id="conflict-icon-button-tooltip">
+                          <small style={{ fontWeight: 500 }}>
+                            Computed with {coursesWithRating} course
+                            {coursesWithRating === 1 ? '' : 's'}.{' '}
+                            {coursesWithoutRating.length === 1
+                              ? coursesWithoutRating[0]
+                              : `${coursesWithoutRating.slice(0, -1).join(', ')} and ${coursesWithoutRating[coursesWithoutRating.length - 1]}`}{' '}
```

Comment:
@bearsyankees If you want to go down this route, I recommend you take a look at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat